tools/coqdep/lexer.mll

```diff
@@ -187,6 +191,18 @@ and comment = parse
   | _
       { comment lexbuf }
 
+and skip_parenthesized = parse
+  | "(*"
+      { comment lexbuf; skip_parenthesized lexbuf }
+  | "("
+      { skip_parenthesized lexbuf; skip_parenthesized lexbuf }
```

Comment:
Do you really need nesting?

Reply:
We have the `module(inductive(..))` syntax
and maybe more complicated categories in the future